flight/AIs/user_command.py

```diff
@@ -0,0 +1,73 @@
+import threading
+
+from ..drone.drone_controller import DroneController
+from .. import constants as c
+from ... import flightconfig as f
+
+def input_loop(controller):
+    """Simple command line interface to drone controller.
+
+    exit: lands and terminates program
+    land: land <priority>
+    hover: hover <duration> <priority>
+    takeoff: currently not used for safety reasons
+    move: move <FOWARD,BACK,LEFT,RIGHT> <duration> <priority>
```

Comment:
```suggestion
    move: move <FORWARD,BACK,LEFT,RIGHT> <duration> <priority>
```

Reply:
also you have `BACK` here and `BACKWARD` below

Reply:
Thanks. I have also added the command formatting for takeoff, since it is now usable.